src/Uno.Templates/content/unoapp/.template.config/template.json

```diff
@@ -422,9 +422,32 @@
     },
     "http": {
       "displayName": "HTTP",
-      "description": "Register native handlers for http and setup Refit support",
+      "description": "Configure HTTP support in the application",
       "type": "parameter",
-      "datatype": "bool"
+      "datatype": "choice",
+      "defaultValue": "basic",
```

Comment:
```suggestion
```

You don't need a `defaultValue` here. It will conflict. It is handled by the `presetHttpDefault`, and it would conflict with the TemplateWizard.json

https://github.com/unoplatform/uno.templates/blob/cc2484fbeb724c6df16e131a8ce5a37a6437f249/src/Uno.Templates/content/unoapp/.template.config/template.json#L1229-L1246